action-dynamic.Rmd

```diff
@@ -416,7 +416,7 @@ ui <- fluidPage(
         choices = c("normal", "uniform", "exponential")
       ),
       numericInput("n", "Number of samples", value = 100),
-      parameter_tabs,
+      parameter_tabs
```

Comment:
Please leave this; it makes it much easier to later rearrange the components.

Reply:
Noted, will update accordingly.

Do you have a preference for how you'd like these updates handled? For example:

* Adding another commit with trailing comma left as-is and then leaving a comment in the text for the `%||%` explanation in the `jamespooley:patch-16` for later squash into single commit when merged; vs.
* Closing this PR and starting from scratch